content/chainguard/chainguard-images/vulnerabilities-compared.md

```diff
@@ -1,11 +1,11 @@
 ---
 title: "Comparison of Vulnerabilities in Container Images"
-linktitle: "Images Compared"
```

Comment:
We are going to delete this page and redirect it to the vulnerabilities landing page: https://edu.chainguard.dev/chainguard/chainguard-images/vuln-comparison/

Do you want to do that in this PR? Just remove and update the nginx config